### ISSUE TITLE
Go client adds additional quotes to `DefaultEndpoint`

### ISSUE THREAD

**Issue by nikolasburk** [prisma/prisma#3987]
When generating the Prisma client in Go, it generates the `DefaultEndpoint` like so:

```go
var DefaultEndpoint = ""https://eu1.prisma.sh/nikolas/prisma-test/ytr""
```

When the client is used then, Go throws this error:

```
prisma-client/prisma.go:66:25: syntax error: unexpected https after top level declaration
```

This can be fixed by removing the extra pair of quotes:

```go
var DefaultEndpoint = "https://eu1.prisma.sh/nikolas/prisma-test/ytr"
```

You can reproduce this by following the instructions in any of the Go examples or by just using `prisma init` and generating the Go client.

**Comment by bmschwartz** [comment]
It may be on older versions but I experienced this on 1.25.7

**Comment by divyenduz** [comment]
This is available in the latest `stable` - `1.26.6`, please install it via `npm install -g prisma`.
